plugins/myfitnesspal.py

```diff
@@ -2,21 +2,33 @@
 import math
 import requests
 from bs4 import BeautifulSoup
+from dateutil import parser
 
 from cloudbot import hook
 
-scrape_url = "http://www.myfitnesspal.com/food/diary/{}"
+scrape_url = "http://www.myfitnesspal.com/food/diary/{}?date={}"
 
 
 @hook.command('mfp', 'myfitnesspal')
 def mfp(text, bot):
-    """<user> - returns macros from the MyFitnessPal food diary of <user>"""
-    request = requests.get(scrape_url.format(text))
+    """
+    <user> [date]- returns macros from the MyFitnessPal food diary of <user>
+                   optionally, specify [date] to retrieve that day's diary
```

Comment:
The core will only grab the first line of the docstring for command documentation, you'll want to combine these lines.